openedx/features/portfolio_project/views.py

```diff
@@ -17,6 +17,13 @@ class GenericTabView(CourseTabView):
     """
     Provides a blank page that acts as its own tab in courseware for displaying content.
     """
+
+    def uses_bootstrap(self, request, course, tab):
+        """
+        Returns true
```

Comment:
FYI: I think we are moving toward not adding doc strings that are restating the obvious.